src/color-cell.scss

```diff
@@ -0,0 +1,87 @@
+// // Copyright (c) 2024 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// // See LICENSE in the project root for license information.
+
+// // ---------------------------------------------------------------------
+
+/* stylelint-disable selector-max-class, selector-max-combinators, selector-max-compound-selectors */
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+@mixin in-range-pseudo-elem {
+  background-color: var(--ds-auro-calendar-cell-in-range-color);
+}
+
+:host {
+  ::slotted([slot^="date_"]) {
+    color: var(--ds-auro-calendar-cell-price-text-color);
+  }
+
+  ::slotted([slot^="date_"][highlight]) {
+    --ds-auro-calendar-cell-price-text-color: var(--ds-color-text-success-default, #{$ds-color-text-success-default});
+  }
+}
+
+.day {
+  border-color: var(--ds-auro-calendar-cell-border-color);
+  background-color: var(--ds-auro-calendar-cell-container-color);
+  color: var(--ds-auro-calendar-cell-text-color);
+
+  &.selected {
+    --ds-auro-calendar-border-color: var(--ds-color-border-ui-active-default, #{$ds-color-border-ui-active-default});
+    --ds-auro-calendar-cell-container-color: var(--ds-color-container-ui-primary-active-default, #{$ds-color-container-ui-primary-active-default});
+    --ds-auro-calendar-cell-text-color: var(--ds-color-text-primary-inverse, #{$ds-color-text-primary-inverse});
+
+    box-shadow: 0 .1px 5px var(--ds-auro-calendar-cell-boxshadow-color-one), 0 5px 10px var(--ds-auro-calendar-cell-boxshadow-color-two);
+
+    &:hover {
+      --ds-auro-calendar-cell-container-color: var(--ds-color-container-primary-default, #{$ds-color-container-primary-default});
+      --ds-auro-calendar-cell-border-color: var(--ds-color-border-ui-focus-default, #{$ds-color-border-ui-focus-default});
+
+      box-shadow: unset;
+    }
+
+    &:not(:hover) {
+      --ds-auro-calendar-cell-price-text-color: var(--ds-color-text-primary-inverse, #{$ds-color-text-primary-inverse});
+    }
+  }
+
+  &:hover {
+    --ds-auro-calendar-cell-container-color: var(--ds-color-container-ui-secondary-hover-default, #{$ds-color-container-ui-secondary-hover-default});
+    --ds-auro-calendar-cell-text-color: var(--ds-color-text-primary-default, #{$ds-color-text-primary-default});
+  }
+
+  &.disabled {
+    --ds-auro-calendar-cell-text-color: var(--ds-color-text-ui-disabled-default, #{$ds-color-text-ui-disabled-default});
+    --ds-auro-calendar-cell-container-color: var(--ds-color-container-primary-default, #{$ds-color-container-primary-default});
+  }
+
+  &.inRange {
+    &:before {
+      @include in-range-pseudo-elem;
+    }
+  }
+
+  &.rangeDepartDate {
+    &:before {
+      @include in-range-pseudo-elem;
+    }
+  }
+
+  &.rangeReturnDate,
+  &.lastHoveredDate {
+    :before {
+      @include in-range-pseudo-elem;
+    }
+  }
```

Comment:
Seems like all three of these selector groups (line 59, 65, 71-22) could all be combined since the nested code is the same.